plugin/trino-hive/src/test/java/io/trino/plugin/hive/util/TestAcidTables.java

```diff
@@ -450,6 +450,34 @@ public void testDeleteDeltaSubdirPathGeneration()
         assertThat(deleteDeltaSubdirPath).isEqualTo("delete_delta_0000013_0000013_0005");
     }
 
+    @Test
+    public void testSkippingSubDirectories()
```

Comment:
Please add an equivalent test to `TestHiveAcidUtils` to verify that the Hive code has the same behavior.